### DISCUSSION TITLE
Review : HoneyGain

### DISCUSSION THREAD

**Discussion by OlivierGaland** [OlivierGaland/CashFactory#8] [Announcements]
Here is my current dashboard following 1 month of use of honeygain application :

![image](https://user-images.githubusercontent.com/26048157/151466665-06dfda28-98e7-427f-85ac-9bc6226c8fba.png)

Some info on the devices : I actually run 3 devices. 2 devices are running on my home connection, 1 device is running on my 4G connection. I notice I have half trafic on my 4G connection (1 device), so I may try to run a second instance on my 4G to check if it double my trafic. Note that honeygain accept only 2 devices max per IPs.

When I registered I was awarded 5000 crédits ($5) as I was using a referral link. I'm currently at 12300 crédits, so no payout yet (need 20000). I was very confident at first as I was gaining 600 crédits (0.6$) per day which means almost $20/month, then the week later it went to 250 then to 100 ... I don't know why (please confirm if you have same issue).

So this app had at first the same profit as earnapp but now is much below average, considering $20 is the minimal payout I will need a couple months to expect reaching the payout limit ... I'm hoping this is a temporary situation and the traffic will raise in the future.

Referal link : https://r.honeygain.me/GALAN2C368

Don't hesitate to post any additionnal information on this application.

**Comment by Micka-R** [comment]
![image](https://user-images.githubusercontent.com/68512105/160095329-84eb08a7-3158-4b02-9aad-b017d1011181.png)
I have the same issue, the traffic rate was 300 a day with one server + android phone on 4g 27/7, then it dropped to 83 a day.

**Comment by OlivierGaland** [comment]
![image](https://user-images.githubusercontent.com/26048157/160200637-dc988af8-17b8-4760-827f-ad59a9c482c3.png)

Hello, I can confirm after almost 3 months that the earning are still lower than the first days : between 80 and 200 (around 140 average) ... I don't know why the first two weeks they were much higher then dropped.

Note I'm now using the JMPT wallet (it offer +50% credits). I also noticed that the standard wallet and JMPT wallet are not mergable.
This means that you must first reach the $20 payout on the standard wallet then switch to JMPT (so you can get the earned $, otherwise it will stay in the standard wallet)

I confirm that the payout work, I got my $20 yesterdays (so it tooks almost 3 months ...), payment I choosed was bitcoin.

For some who want to switch to JMPT, please note that you need an intermediate level in crypto. JMPT token is not listed on the main exchange, so it means that you need to use pancakeswap to trade against a well known crypto (seems to be BUSD or BNB), then send it to your exchange to convert into EUR or USD (or any other fiat)

**Comment by OlivierGaland** [comment]
Here is my dashboard after a few days using JMPT mode, revenue have increased a lot. I will check if it stay on this level.

![image](https://user-images.githubusercontent.com/26048157/160951085-4ae29b82-e59c-40af-a3e2-d59cd8d3421f.png)

**Comment by nitrup** [comment]
For me :)
![Capture d’écran 2022-03-31 114707](https://user-images.githubusercontent.com/35567280/161027440-e25aa31f-a6da-4f79-b7da-20b6f976988b.png)

**Comment by OlivierGaland** [comment]
Yes that's seems to confirm it is much better to use the jumptask wallet to earn money for Honeygain, there is much more credits added.

**Comment by Micka-R** [comment]
Till now was using 2 instance on one ip to double the gains. Now it seems one instance by ip is the maximum can someone confirm that plz?

edit with my 4g I can have 2 instance but on my home network I am limited to one... weird.

**Reply by juliankowalski** [reply]
https://support.honeygain.com/hc/en-us/articles/360011078900-Error-Network-overused

**Comment by OlivierGaland** [comment]
Hello, Yes I confirm i had same issue, it was possible to run 2 instances / ip but now i'm limited to 1 instance / ip.

**Comment by Bidmaster777** [comment]
Hi, did anyone get their IP blacklisted for spam as well by using Honeygain (Cashfactory)??
My 2 IP's did at the same time.... ;-(

**Reply by lyenliang** [reply]
May I know which website added your IP address in their blacklist?
This is what I'm afraid of when using Honeygain.

**Reply by abatyd** [reply]
Try to check your public IP adress with this tool : [https://mxtoolbox.com/blacklists.aspx](url)

**Comment by lyenliang** [comment]
I have created a [HoneygainBot](https://github.com/lyenliang/HoneygainBot) that automatically press the "Open Lucky Pot" button every day. You can checkout how to use it here: https://github.com/lyenliang/HoneygainBot

**Comment by Sigmoidas** [comment]
Anyone had any success running Honeygain on Raspberry Pi 3 arm32 hardware?
